snap/snapcraft.yaml

```diff
@@ -65,9 +65,12 @@ apps:
     slots:
       - wayland
     plugs:
+      - desktop-launch
+      - fonts
       - login-session-control
-      - x11
       - hardware-observe
+      - opengl
```

Comment:
Do we need to plug opengl? Does that give us anything the wayland slot does not?

Reply:
I've now dropped them from the global `plugs:`.